store/postgres/src/notification_listener.rs

```diff
@@ -25,6 +26,15 @@ lazy_static! {
                 )))
             )
             .unwrap_or(Duration::from_secs(300));
+    static ref DB_NOTIFICATION_BROADCAST_TIMEOUT: Duration =
+        env::var("GRAPH_DB_NOTIFICATION_BROADCAST_TIMEOUT")
+            .ok()
+            .map(
+                |s| Duration::from_secs(u64::from_str(&s).unwrap_or_else(|_| panic!(
+                    "failed to parse env var GRAPH_DB_NOTIFICATION_BROADCAST_TIMEOUT"
+                )))
+            )
+            .unwrap_or(Duration::from_secs(600));
```

Comment:
I actually think the default should be reduced to 60s - for chain head updates, we already start polling after 30s, and for subscriptions :shrug:

Reply:
Makes sense, done